### PR TITLE
Add present operator

### LINKED ISSUE #40: Support conditions that need to be agnostic of undefined/null
Currently we have the option to compare a value against `null`, or to use the `UNDEFINED` comparison expression.

It would be nice if we had a way to cover both cases easily. 

For example, the following condition:

```
"condition": [
    "NOR",
    [
      "UNDEFINED",
      "$Loss1Date"
    ],
    [
      "==",
      "$Loss1Date",
      null
    ]
  ]
```

Say it's possible for `Loss1Date` to be on the page but left empty, or maybe conditionally not shown at all, or filled out, and then emptied by the user. By the time it comes to evaluate the condition it might be either `null` or `undefined`. It would be nice if we can capture that in a single expression.

Some options:
* `UNDEFINED_OR_NULL` expression
* Updating `UNDEFINED` to check for null as well
* `EMPTY` / `NOT_EMPTY` expression
* `NIL` / `NOT_NIL` expression

### PR DESCRIPTION
**Pull request by lilyvanoekel**
Add present operator

Closes #40

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -387,6 +388,22 @@ engine.evaluate(['UNDEFINED', 'RefA'], { RefA: undefined }); // true
 engine.evaluate(['UNDEFINED', 'RefA'], { RefA: 10 }); // false
 ```
 
+#### Present
+Returns FALSE when the operand is UNDEFINED or NULL.
```
Comment (davidhorak) [inline readme.md:392]: ~Returns~ Evaluates as FALSE ...

File: src/__test__/unit/index.test.ts [hunk 2]
```diff
@@ -166,6 +166,33 @@ describe('Condition Engine', () => {
...
+        '$Name'
+      ],
+      inputs: [{
+        data: {
+          Name: undefined
+        },
+        expected: false
+      }, {
+        data: {
+          Name: 'David'
```
Comment (davidhorak) [inline src/__test__/unit/index.test.ts:181]: nice :D

File: readme.md (hunk 1 above)
Comment (lilyvanoekel) [reply readme.md]: 3eee647c81ce2d52620485e426f55e797ad7ca1d

**Comment by rthreei** [comment]
This is nice 👍

**Pull request merged**